pallets/nicks/src/migrations.rs

```diff
@@ -0,0 +1,65 @@
+use super::*;
+pub use pallet::*;
+use sp_std::prelude::*;
+use frame_support::{
+	weights::Weight,
+	pallet_prelude::*,
+	traits::{Get, StorageVersion},
+	storage::migration,
+};
+
+pub fn migrate<T: Config>() -> Weight {
```

Comment:
Refer to:
https://github.com/paritytech/substrate/blob/a1fc5cff2ad979697384b24dd52671ccba0bf128/frame/contracts/src/migration.rs#L28